src/util/generatePreview.js

```diff
@@ -58,6 +65,7 @@ class PreviewGenerator {
     if (options.propagateErrorsToParent) {
       this._addErrorHandling();
     }
+    this._addAlertHandling();
```

Comment:
Added a new option to the PreviewGenerator, whether we are in a popout view or not. If false, then we use sweetalert.